src/node.ts

```diff
@@ -284,6 +307,22 @@ class Node extends VEventTarget {
   log = (...args: unknown[]): void => {
     this.dispatchEvent(new VEvent('log', { detail: { args } }));
   };
+
+  /**
+   * Method that sets resetState to false
```

Comment:
Don't just describe what this method's code does. Explain the methods purpose!
"The action code must call this method, if the node has to maintain state until the next cycle."